openapi-specs/threat-vault/ThreatPrevention.yaml

```diff
@@ -1133,6 +1185,195 @@ paths:
         - X-API-KEY: []
       summary: Request Threat Signature Release History
       tags: [Threat Prevention]
+
+
+  /edl:
+    get:
+      operationId: edl
+      description: |
+        A GET request to retrieve the predefined external dynamic lists by name and version.
+      parameters:
+      - name: name
+        in: query
+        required: false
+        schema:
+          type: string
+          enum:
+            - panw-known-ip-list
+            - panw-highrisk-ip-list
+            - panw-torexit-ip-list
+            - panw-bulletproof-ip-list
+        description: |
+          Provides the EDL by the name.
+    
+          ---
+          ```
+          curl -v -H 'X-API-KEY: API_KEY' \
+            'https://api.threatvault.paloaltonetworks.com/service/v1/edl?name=panw-known-ip-list&version=latest'
+          ```
+
+      - name: ipaddr
+        in: query
+        required: false
+        schema:
+          type: string
+        description: |
+          Provides the EDL of the IP address.
```

Comment:
Provides the EDLs containing the IP address.

Note: I think an IP may be in > 1 EDL.